packages/signing-utils/src/signing-clients/index.ts

```diff
@@ -0,0 +1,62 @@
+import type { ConnectConfig } from 'ssh2';
+
+import * as path from 'path';
+import { SSHClient } from '../ssh-client';
+import { LocalSigningClient } from './local-signing-client';
+import { RemoteSigningClient } from './remote-signing-client';
+
+export { LocalSigningClient } from './local-signing-client';
+export { RemoteSigningClient } from './remote-signing-client';
+
+export type SigningClientOptions = {
+  rootDir: string;
+  signingScript: string;
+  signingMethod: 'gpg' | 'jsign';
```

Comment:
nit: we can have a type for this to avoid repetition.

Reply:
fixed